Assets/Script/PlayerController.cs

```diff
@@ -28,7 +28,12 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        var input = Input.GetAxis("Horizontal") * speed;
+      HandlerMovement();
+    }
+
+    private void HandlerMovement()
```

Comment:
typo, it would be `HandleMovement`  or `MovementHandler` , I like more the later following the C# [convention](https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-delegates-and-event-handlers#event-handler-naming-conventions) although we are not using Event Handlers